framework/auth/campaigns.py

```diff
@@ -74,7 +74,8 @@ def get_campaigns():
         except (NoResultsFound or QueryException or ImproperConfigurationError) as e:
             logger.warn('An error has occurred during campaign initialization: {}', e)
 
-    CAMPAIGNS_LAST_REFRESHED = datetime.utcnow()
+        CAMPAIGNS_LAST_REFRESHED = datetime.utcnow()
```

Comment:
`CAMPAIGNS_LAST_REFRESHED` should be only updated when `CAMPAIGNS` are recalculated.